### PR TITLE
INTYGFV-14467: Distinguish between mobile and desktop bank_id login

### PR DESCRIPTION
**Pull request by SamuelTilly**
https://github.com/sklintyg/frontend/pull/484
https://github.com/sklintyg/webcert/pull/767

### CODE REVIEW

File: support/src/main/java/se/inera/intyg/common/support/facade/model/user/LoginMethod.java [hunk 1]
```diff
@@ -19,5 +19,5 @@
 package se.inera.intyg.common.support.facade.model.user;
 
 public enum LoginMethod {
-    BANK_ID, SITHS, FAKE
+    BANK_ID, SITHS, FAKE, BANK_ID_MOBILE
```
Comment (mww-sopra-steria) [inline support/src/main/java/se/inera/intyg/common/support/facade/model/user/LoginMethod.java:22]: What is the difference between BANK_ID and BANK_ID_MOBILE for the purpose of signing?

**Review by mww-sopra-steria — CHANGES_REQUESTED** [review]
Just a question concerning the different BANK_ID LoginMethods. Not sure that anything has to be done before merging, but just want to raise this first to understand.

File: support/src/main/java/se/inera/intyg/common/support/facade/model/user/LoginMethod.java (hunk 1 above)
Comment (SamuelTilly) [reply support/src/main/java/se/inera/intyg/common/support/facade/model/user/LoginMethod.java:22]: The only difference is really only some copy. This is taken directly from the old client https://github.com/sklintyg/frontend/pull/484/files#diff-b86c34ac30f1f81a6658a0ebf86ab6b77ec35455c92490732eb83ecb0fd49addR13-R17

Comment (mww-sopra-steria) [reply support/src/main/java/se/inera/intyg/common/support/facade/model/user/LoginMethod.java:22]: OK! Then I understand. We want to give different messages to the user depending if they are using MOBILE or DESKTOP BANKID.

**Review by mww-sopra-steria — APPROVED** [review]
Approved!

**Pull request merged**